### DISCUSSION TITLE
Concrete Strength with Random Forest

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#867] [Q&A] [answered]
malam ka, mau nanya sama hasil capstone dulu disini https://rpubs.com/rafiff23/1181702 saya punya beberapa pertanyaan general 

yang pertama adalah jika random forest itu di tuned dengan random search attributes mana yang di tuned?
dan yang kedua adalah apakah ada specific features mempengaruhi concrete strength predictions? dan apakah ada anomalies observed dalam analysis nya?

**Comment by ranydwi** [comment, marked as answer]
Dear Pak @rafiff23 

Thank you untuk pertanyaannya dan kesediaannya untuk menunggu. 
- Pertanyaan yang mengenai anomalies observed apakah asumsi adanya observasi-observasi anomali atau apa ya? Boleh di elaborate lagi Pak?
- Untuk pertanyaan _Apakah ada specific features mempengaruhi concrete strength predictions?_  Jawaban pertanyaan ini apabila kita menggunakan metode random forest sebetulnya tidak memungkinkan, hanya varImp (Variable Importance) yang bisa didapatkan. Karena jawaban causal impact hanya bisa didapat dari metode linear regresi Pak

Regards
Rany.

**Comment by ramdhinafinita** [comment]
Halo @rafiff23, untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
